docs/basics/migrating-to-webpacker.md

```diff
@@ -0,0 +1,67 @@
+1. Add `gem "webpacker_lite"` to the Gemfile, run bundle
+2. create /config/webpack/paths.yml with the following content
+
+  ```
+  default: &default
+    config: client/webpack
```

Comment:
This will probably change due to https://github.com/shakacode/react-webpack-rails-tutorial/pull/387#pullrequestreview-36677413 and https://github.com/shakacode/react_on_rails/pull/834